spring-kafka/src/main/java/org/springframework/kafka/core/KafkaStreamsCustomizer.java

```diff
@@ -18,6 +18,8 @@
 
 import org.apache.kafka.streams.KafkaStreams;
 
+import org.springframework.kafka.config.StreamsBuilderFactoryBean;
```

Comment:
I would say this `KafkaStreamsCustomizer` is part of the configuration as well.
Feels organic to move it into the `config` package as well.

Reply:
Agreed; it crossed my mind too. Will do.